community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+8. Run the Flow_BuildAndValidate_${RELEASE} job.
+
+    a. Select `stable` for **HALYARD_RELEASE_TRACK**.
+    
+    b. This will automatically update the [changelog gist](https://gist.github.com/spinnaker-release/4f8cd09490870ae9ebf78be3be1763ee)
+    on GitHub.
+    
+9. Add the new Flow_BuildAndValidate_${RELEASE} job to the public
+[Build Statuses page](https://www.spinnaker.io/community/contributing/build-statuses/#nightly-and-release-integration-tests).
+Ping #dev with some version of this message, including a link to the correct
+section of the changelog gist:
+
+> The release window for Spinnaker $VERSION is now open!  This means that
+> release branches have been cut from master and those branches are only
+> accepting fixes for existing features.  Please contact $YOUR_NAME
+> (slack: $YOUR_SLACK_ID, github: $YOUR_GITHUB_ID, or email: $YOUR_EMAIL) if you
+> would like a fix cherry-picked into the release or you would like to highlight
+> a specific fix or feature in the release’s changelog. If you’d like to jog
+> your memory of everything to be released with Spinnaker $VERSION, see the raw
+> changelog here: $LINK_TO_CHANGELOG.
+
+10. Share a curated changelog with any partners from the community who wants to
+add notes.
+
+11. When the Flow_BuildAndValidate_${RELEASE} job passes, ping #dev with a
+message that release candidate is now validated and can be tested by running:
+
+```
+hal config version edit --version ${RELEASE_BRANCH}-latest-validated
+```
+
+## Second Monday of the release window
+
+1. Check for any PRs waiting to be [cherry-picked](https://github.com/search?q=org%3Aspinnaker+label%3Acherry-pick+state%3Aopen+type%3Apr).
+(You can further restrict the query by adding a constraint like +base:release-1.18.x to the URL.)
+
+2. Rerun the Flow_BuildAndValidate_${RELEASE} job and get a green build.
+
+3. Create a new gist for this release.
+
+    a. Log into GitHub as spinnaker-release and create a new gist to hold the
+    release notes for this release branch.
+    
+    b. The description should be “Spinnaker 1.nn.x Release Notes” (e.g.,
+    Spinnaker 1.18.x Release Notes). The gist will eventually have a separate
+    file with the release notes for each patch release on this branch.
+
+    c. Add a file 1.nn.0.md (e.g., `1.18.0.md`) to hold the release notes for
+    the new release.
+    
+    d. Copy the changes for this release from the raw build changelog to the new
+    1.nn.0.md file.
+
+    e. Add the notes from the curated changelog to the top of the gist,
+    formatting them for Markdown ([sample 1.nn.0 release notes](https://gist.github.com/spinnaker-release/cc4410d674679c5765246a40f28e3cad)).
+
+4. Run Publish_SpinnakerRelease:
+
+    a. **Spinnaker Version** is "1.nn.0" (replacing nn with the version number).
+    
+    b. **Spinnaker Release Alias** should be the name of a Netflix original TV
+    show converted to a camel-case alphanumeric string
+    (e.g., "MichaelBoltonsBigSexyValentinesDaySpecial").
```

Comment:
The last release was not converted to camel case and apparently nothing blew up, so should we remove that suggestion? It's "Gilmore Girls A Year in the Life"

Reply:
Well, if only you knew the pain I caused by initially calling it "Gilmore Girls: A Year in the Life"...turns out we interpolate it directly into YAML...

Reply:
Well, I have just the PR to fix that, by moving the publish code into spinrel. Hopefully ready before the 1.20.0 release.

Reply:
Sure, I will also update the example to "Gilmore Girls A Year in the Life" so that release name is immortalized.